opentracing-api/src/main/java/io/opentracing/ThreadLocalScheduler.java

```diff
@@ -0,0 +1,70 @@
+package io.opentracing;
+
+/**
+ * ThreadLocalScheduler is a trivial Scheduler implementation that relies on Java's thread-local storage primitives.
+ *
+ * @see Scheduler
+ * @see Tracer#scheduler()
+ */
+public class ThreadLocalScheduler implements Scheduler {
```

Comment:
My only issue with it being here would be that there aren't currently any implementations in this module other than for tags. Since we are making a major version change on the library I don't feel that it would be unexpected as a dev to have to create/find a new component